dev/io.openliberty.org.testcontainers/cache/projects

```diff
@@ -175,7 +175,7 @@ com.ibm.ws.transaction.hadb_fat.sqlserver.2
 com.ibm.ws.transaction.hadb_fat.sqlserver.lease
 com.ibm.ws.transaction.hadb_fat.sqlserver.retriablecodes
 com.ibm.ws.wsat.migration_fat.6
-com.ibm.ws.wsat.recovery_fat.multi.6
+com.ibm.ws.wsat.recovery_fat.multi.8
```

Comment:
The changes in this file do not look related. I would drop them to avoid risk of a conflict with another PR.